distributed/client.py

```diff
@@ -4741,7 +4741,8 @@ def default_client(c=None):
         )
 
 
-def ensure_default_get(client):
+def ensure_default_client(client):
+    "Ensures the client passed as argument is set as the default."
```

Comment:
Style nit: Single and triple-quote Python strings have slightly different behavior. Most folks use triple quotes everywhere for docstrings

```suggestion
    """Ensures the client passed as argument is set as the default"""
```